modules/features/endofyear/src/main/java/au/com/shiftyjelly/pocketcasts/endofyear/stories/EndOfYearStoriesDataSource.kt

```diff
@@ -13,15 +13,19 @@ class EndOfYearStoriesDataSource @Inject constructor(
         return combine(
             endOfYearManager.getTotalListeningTimeInSecsForYear(YEAR),
             endOfYearManager.findListenedCategoriesForYear(YEAR),
+            endOfYearManager.findListenedNumbersForYear(YEAR),
             endOfYearManager.findRandomPodcasts(),
-        ) { listeningTime, listenedCategories, podcasts ->
+        ) { listeningTime, listenedCategories, listenedNumbers, podcasts ->
             val stories = mutableListOf<Story>()
 
             listeningTime?.let { stories.add(StoryListeningTime(it)) }
             if (listenedCategories.isNotEmpty()) {
                 stories.add(StoryListenedCategories(listenedCategories))
                 stories.add(StoryTopListenedCategories(listenedCategories))
             }
+            if (listenedNumbers.numberOfEpisodes > 0 && listenedNumbers.numberOfPodcasts > 0) {
```

Comment:
I konw the strings are just placeholders, but noting that if we're going to display this screen with 1 episode or 1 podcast, we'll need to have strings to accommodate those numbers. It might be better to just not show this screen if the number of episodes is 1 since that doesn't really provide any value.

Reply:
Good point, I updated the condition in f6675c6.